### PR TITLE
Update scripts

### PR DESCRIPTION
**Pull request by josephignace**
Table 2 headers updated.
Mixed dementia added to common_variables.py. and plot_labels.csv Day 1 added to active_analyses.R
Venn diagram script include source contribution for mixed dementia (lines will be improved on a different PR).

### CODE REVIEW

File: analysis/create_project_actions.R [hunk 1]
```diff
@@ -431,7 +431,7 @@ actions_list <- splice(
                  "table2_vax",
                  "table2_unvax"),
     moderately_sensitive = list(
-      table2_output_rounded = glue("output/table2_output_rounded.csv")
+      table2_output_midpoint6_derived = glue("output/table2_output_midpoint6_derived.csv")
```
Comment (venexia) [inline analysis/create_project_actions.R:434]: There are no specific recommendations around filenames as far as I can see, just variable names. Please pick one filename type and be consistent though so either '_rounded' or '_midpoint6' or '_midpoint6_derived'.

File: analysis/model/make_table2_output.R [hunk 2]
```diff
@@ -17,44 +17,44 @@ df <- data.frame(name = character(),
...
+  table2$exposed_events_midpoint6 <- roundmid_any(as.numeric(table2$exposed_events), to=threshold)
+  table2$day0_events_midpoint6 <- roundmid_any(as.numeric(table2$day0_events), to=threshold)
+  table2$total_exposed_midpoint6 <- roundmid_any(as.numeric(table2$total_exposed), to=threshold)
+  table2$sample_size_midpoint6 <- roundmid_any(as.numeric(table2$sample_size), to=threshold)
   
   # Recalculate total columns --------------------------------------------------
   print('Recalculate total columns')
   
-  table2$total_events_rounded <- table2$exposed_events_rounded + table2$unexposed_events_rounded
+  table2$total_events_midpoint6 <- table2$exposed_events_midpoint6 + table2$unexposed_events_midpoint6
```
Comment (venexia) [inline analysis/model/make_table2_output.R:48]: This is a derived variable and should be named to reflect this.

**Review by venexia — CHANGES_REQUESTED** [review]

File: analysis/create_project_actions.R (hunk 1 above)
Comment (josephignace) [reply analysis/create_project_actions.R:434]: Your instructions on slack were unclear, you only requested to change _rounded with _midpoint6 and _midpoint6_derived. The changes I made work, and there has been no issue with dummy data and real data (analysis currently running on the server).

File: analysis/model/make_table2_output.R [hunk 3]
```diff
@@ -45,15 +45,15 @@ for (cohort in c("prevax","vax","unvax")) {
   # Recalculate total columns --------------------------------------------------
   print('Recalculate total columns')
   
-  table2$total_events_midpoint6_derived <- table2$exposed_events_midpoint6 + table2$unexposed_events_midpoint6
+  table2$total_events_midpoint6_redacted <- table2$exposed_events_midpoint6 + table2$unexposed_events_midpoint6
```
Comment (venexia) [inline analysis/model/make_table2_output.R:48]: This was correct before these changes. It should be `total_events_midpoint6_derived`.

Comment (venexia) [reply analysis/model/make_table2_output.R:48]: Apologies - I see this was a typo in the minutes from our meeting.

File: analysis/descriptives/table2.R [hunk 4]
```diff
@@ -127,4 +128,4 @@ table2[,setdiff(colnames(table2),c("name","cohort","exposure","outcome","analysi
 # Save Table 2 -----------------------------------------------------------------
 print('Save rounded Table 2')
 
-write.csv(table2, paste0("output/table2_",cohort,"_rounded.csv"), row.names = FALSE)
\ No newline at end of file
+write.csv(table2, paste0("output/table2_",cohort,"_midpoint6.csv"), row.names = FALSE)
```
Comment (venexia) [inline analysis/descriptives/table2.R:131]: If you haven't run Table 2 yet, you can fix the midpoint6 rounding here at the source and use `make_other_output.R` to create the Table 2 output. The file `make_table2_output.R` is a hack if you have already run Table 2 (which takes a long time to run) but need to fix the midpoint6 rounding for outputting. This was discussed on Slack [here](https://grp-ehr.slack.com/archives/C03RWTCBVT9/p1699019578661349).

**Review by venexia — CHANGES_REQUESTED** [review]

File: analysis/descriptives/table2.R (hunk 4 above)
Comment (josephignace) [reply analysis/descriptives/table2.R]: Changed script.

Comment (josephignace) [reply analysis/descriptives/table2.R]: Point 1 is not clear. Point 2 done. Point 3 cannot be done due to not clear instructions.

**Pull request closed without merging**